arquillian-rusheye/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.arquillian.universe</groupId>
+    <artifactId>arquillian-universe-parent</artifactId>
+    <version>1.1.13.5-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-rusheye</artifactId>
+  <name>Arquillian Rusheye</name>
+  <packaging>pom</packaging>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.arquillian.rusheye</groupId>
+        <artifactId>rusheye-impl</artifactId>
+        <version>${version.rusheye}</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.arquillian.rusheye</groupId>
+      <artifactId>rusheye-impl</artifactId>
+    </dependency>
+  </dependencies>
```

Comment:
For me is more that let's do how it is done before so at least code has some consistency, and at the end it is just 4 lines of code.